src/TileViewer.cpp

```diff
@@ -94,10 +92,15 @@ TileViewer::TileViewer(QWidget* parent)
             this, &TileViewer::imageMouseOver);
     connect(imageWidget, &VramTiledView::imageClicked,
             this, &TileViewer::displayCharInfo);
+
+    VDPDataStore::instance().refresh();
 }
 
 void TileViewer::VDPDataStoreDataRefreshed()
 {
+    const auto* vram = VDPDataStore::instance().getVramPointer();
+    imageWidget->setVramSource(vram);
+
     if (useVDPPalette->isChecked()) {
         imageWidget->setPaletteSource(VDPDataStore::instance().getPalettePointer());
```

Comment:
This commit itself is fine. Thanks.

But I have a comment on the coding style in the debugger in general.
In these 5 lines I already see `VDPDataStore::instance()` repeated twice. If I look at the full code, I see many more repeats (within the same methods). It would be nice to clean this up (in a separate commit).